themes/af-magic.zsh-theme

```diff
@@ -27,7 +27,7 @@ else
 fi
 
 # git settings
-ZSH_THEME_GIT_PROMPT_PREFIX="$FG[075](branch:"
+ZSH_THEME_GIT_PROMPT_PREFIX="($FG[078]branch:"
 ZSH_THEME_GIT_PROMPT_CLEAN=""
 ZSH_THEME_GIT_PROMPT_DIRTY="$my_orange*%{$reset_color%}"
-ZSH_THEME_GIT_PROMPT_SUFFIX="$FG[075])%{$reset_color%}"
+ZSH_THEME_GIT_PROMPT_SUFFIX="$FG[078])%{$reset_color%}"
```

Comment:
I want to make parenthesis blue while only branch part is green. 
I checked my pull request works as I expected and you can see it works on the screenshot I uploaded.

As I know, color directives( ex. $FG[078]) affect to characters next to it, so I don't understand `$FG[078]%{$reset_color%}` does mean anything. I think the code set next characters green and reset the color to white immediately.

please check that and let me know if I'm wrong, thanks!